.projenrc.ts

```diff
@@ -152,8 +152,22 @@ const fwkProject = new awscdk.AwsCdkConstructLibrary({
       target: "ES2021"
     }
   },
+
+  eslintOptions: {
+    dirs: ['src'],
```

Comment:
Do we also want to enable it for tests?

Reply:
I initially did it but it modifies the [tasks.json](https://github.com/awslabs/aws-data-solutions-framework/blob/5b635bdd23014aba24dff48d51998d5f0d3e2d7b/framework/.projen/tasks.json#L77) and add `test` twice...

Reply:
as you can see test is already there...